README.md

```diff
@@ -1,4 +1,6 @@
 easy-datacite
 =============
 
+[![Build Status](https://travis-ci.org/DANS-KNAW/easy-datacite.svg?branch=master)](https://travis-ci.org/DANS-KNAW/easy-datacite)
```

Comment:
I guess that in an archetype you have ways to insert the project name into a string somehow. 
And non-public projects should be kept to minimum, most likely the exception...